### PR TITLE
orders page ui changes

### PR DESCRIPTION
**Pull request by celwell**

### CODE REVIEW

File: src/dashboard_cljs/orders.cljs [hunk 1]
```diff
@@ -722,39 +722,51 @@
...
-                         (subscription-id->name (:subscription_id @order))
-                         " Plan")])]
-                [:h5 [:span [GoogleMapLink
-                             (:address_street @order)
-                             (:lat @order)
-                             (:lng @order)] ", " (:address_zip @order)]]
-                ;;  email
-                [:h5 [Mailto (:email @order)]]
-                ;;  phone number
-                [:h5  [TelephoneNumber (:customer_phone_number @order)]]]
```
Comment (jborden) [inline src/dashboard_cljs/orders.cljs:749]: You removed the format that Bruno particularly said he liked.

Comment (celwell) [inline src/dashboard_cljs/orders.cljs:749]: Which part is that?

**Comment by jborden** [comment]
Have you tested that the interface still works properly? I am very hesitant to merge relatively minor changes without knowing it everything still works properly. Also, see the line note I made, you are altering the customer info page with something that Bruno himself requested and pushed me to change.

**Comment by celwell** [comment]
Yes it works. If the change bothers Bruno I'll be sure he knows I did it.

File: src/dashboard_cljs/orders.cljs (hunk 1 above)
Comment (jborden) [inline src/dashboard_cljs/orders.cljs:749]: The 'Customer Info' page was requested.

**Comment by jborden** [comment]
This is the Asana task for the orders page:
https://app.asana.com/0/64277000503113/120738838208147

Changing a key aspect of the user interface will introduce mental overhead. Considering business has picked up, it really isn't a good idea to do it now. 

The most pertinent information for dispatching is highlighted in the old design, which is customer name, address and contact information. This is followed by vehicle information. 

Market -> Zone - Sure, it accurately reflects what it is called. 

Show All -> Past (filter, display) / Current -> Current Orders - Having a complete history in one table could still be useful.  Maybe someone would ask "How many Tire fill ups have been ordered?" or "How many 5Hr deliveries have been placed?".  If we change this, then the user will have to switch between the tables.

File: src/dashboard_cljs/orders.cljs (hunk 1 above)
Comment (celwell) [inline src/dashboard_cljs/orders.cljs:749]: The info is still there though. The heading Customer Info was a misnomer. It included order address. I think the heading become unnecessary anyways, and take up valuable space. Also the two details boxes (Customer & Order) weren't as balanced as they are now, so there was often unnecessary white space on the left side.

**Comment by celwell** [comment]
I see your point, and already considered the concerns you raised so far, but still think we should merge all of these changes. Most of the changes are backed by day-to-day dash users (e.g., jackson, brandon).

**Pull request merged**